microceph/client/client_configs.go

```diff
@@ -83,15 +83,15 @@ func SendUpdateClientConfRequestToClusterMembers(ctx context.Context, s interfac
 	// Get a collection of clients to every other cluster member, with the notification user-agent set.
 	cluster, err := s.ClusterState().Cluster(false)
 	if err != nil {
-		logger.Errorf("failed to get a client for every cluster member: %v", err)
+		clilogger.Errorf("failed to get a client for every cluster member: %v", err)
```

Comment:
is there a reason why the same logger could not be used ? so as to have a cliLogger.